src/style.css

```diff
@@ -0,0 +1,33 @@
+[class^="stars"] {
+  display: flex;
+}
+
+body {
+  margin: 0;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background: url(./images/star.svg);
+  background-position: center !important;
+  background-repeat: no-repeat !important;
+}
+
+.stars__star:last-child {
+  margin: 0;
+}
+
+[class^="stars--"] {
+  display: flex;
+}
+
+.stars--1 > .stars__star:first-child,
+.stars--2 > .stars__star:nth-child(-n + 2),
+.stars--3 > .stars__star:nth-child(-n + 3),
+.stars--4 > .stars__star:nth-child(-n + 4),
+.stars--5 > .stars__star:nth-child(-n + 5) {
+  background: url(./images/star-active.svg);
+  background-position: 50%;

```

Comment:
already added in .stars__star

Reply:
![image](https://user-images.githubusercontent.com/60158880/215762552-42cda9a0-d166-4c89-8a6c-abb872c3403b.png)

i do not know why but (background-position) is not working for active stars

Reply:
sory i find mistake me should to use background-image not background